tests/unit/AoDropdownItem.spec.js

```diff
@@ -0,0 +1,26 @@
+import { shallowMount } from '@vue/test-utils'
```

Comment:
a test that confirms the click handler fires would be a good idea as it is core to the functionality of the dropdown item

Reply:
^ This can be optional but nice to have